feedback/templates/feedback/feedback_thankyou.html

```diff
@@ -0,0 +1,15 @@
+{% extends "minimal.html" %}
```

Comment:
The usual convention is to extend `"<app name>/template.html"`, so here `"feedback/minimal.html"`.

Reply:
Done.